gradle/libs.versions.toml

```diff
@@ -35,6 +35,7 @@ flatbuffers = "24.3.25"
 freemarker = "2.3.33"
 google-findbugs = "3.0.2"
 google-java-allocation-instrumenter = "3.3.4"
+google-rpc-protos="2.61.2"
```

Comment:
ideally 2.64.x so we are supporting proto-java 4 here, but current latest that isnt an RC is 2.63.2